src/main/editDelete/liveProcess.ts

```diff
@@ -0,0 +1,42 @@
+import { Transcription, Word } from 'sharedTypes';
+
+/**
+ * calculateTime calculates the outputStartTimes of a word based on the
+ * current state of all words in a Transcript. It takes into account edits/deletes
+ * @param word word in Transcript
+ * @param i the current index position in the words array
+ * @param words an array of words from Transcript
+ * @returns the input word with an updated outputStartTime
+ *
+ */
+
+const calculateTime = (word: Word, i: number, words: Word[]): Word => {
+  if (!word.deleted) {
```

Comment:
If you flip this and do an early exit, the rest of the logic will be nested one level less deep which will make it easier to read

Reply:
Conditional statements definitely easier to read now with this change. Thanks